src/file-box.ts

```diff
@@ -119,6 +121,7 @@ class FileBox implements Pipeable, FileBoxInterface {
     url      : string,
     name?    : string,
     headers? : HTTP.OutgoingHttpHeaders,
+    md5?     : string,
```

Comment:
There's no need to maintain a **deprecated** method.

Please remove this line because it's deprecated.